pkg/kubernetes/views/route-panel.html

```diff
@@ -5,8 +5,8 @@
             <i class="pficon pficon-route"></i>
         </div>
         <div class="listing-actions">
-            <button title="Delete" class="btn btn-danger btn-delete pficon pficon-delete" disabled></button>
-            <button title="Adjust" class="btn btn-default pficon pficon-edit" disabled></button>
+            <button id="adjust-route-btn" title="Adjust"  class="btn btn-default pficon pficon-edit"  data-toggle="modal" data-target="#adjust-route-dialog"  data-key="{{item.key}}"></button>
```

Comment:
This should not have a an id='adjust-route-btn'. This template is instantiated multiple times per page, and that would result in an invalid page.